test/integration/functions/op.addition.test.ts

```diff
@@ -28,6 +35,8 @@ describe('evaluation of \'+\' like', () => {
       NaN    NaN    = NaN
       NaN    anyNum = NaN
       anyNum NaN    = NaN
+      
+      "2"^^example:int "3"^^example:int = ${int('5')}
```

Comment:
Even though we only do sync for now, this is actually _really_ powerful in itself, by just writing a tiny config extension.

Reply:
Is this a question? Or are you just saying it looks good?

Reply:
No question, just a statement :-)

Reply:
Okay, I'm glad you like it too.